src/MetaMorpho.sol

```diff
@@ -208,14 +208,12 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     /// @inheritdoc IMetaMorphoBase
     function submitTimelock(uint256 newTimelock) external onlyOwner {
         if (newTimelock == timelock) revert ErrorsLib.AlreadySet();
+        if (pendingTimelock.validAt != 0) revert ErrorsLib.AlreadyPending();
```

Comment:
This design is less prone to error for sure, but is worse in terms of UX: you have to batch a `revokePendingTimelock` with a `submitTimelock`

I also think it's fine, but I believe it is the reason we didn't do this earlier

Reply:
I think it can be more problematic for the cap when the curator or owner want to act fast to disable a market and an updated is pending...

Reply:
But they can bundle a revoke + submit so they can essentially act "fast" (within a block)

Reply:
Yes ofc but this must be clear for them, else you can easily lost minutes because of that, so you need to have something in the fronted and in the doc at least to cover this specific case